templates/summit/index.html

```diff
@@ -1,61 +1,62 @@
+  The Ubuntu Summit is a showcase for the innovative and the ambitious. We invite all experts, builders, engineers, tinkerers, 
+  designers and dreamers to join us for an unforgettable three days.
 {% endblock %}
 
 {% block content %}
   <section class="p-strip u-fixed-width">
-    <h1 class="p-heading--display">Ubuntu Summit 25.10</h1>
+    <h1 class="p-heading--display">Ubuntu Summit 26.04</h1>
   </section>
   <section class="p-section">
     <div class="p-section--shallow">
       <div class="row--50-50">
         <hr class="p-rule" />
         <div class="col">
-          <h2 class="p-heading--1">October 23-24</h2>
+          <h2 class="p-heading--1">May 27-28</h2>
           <h3 class="p-heading--2">Online</h3>
         </div>
         <div class="col">
           <p>
-            The Ubuntu Summit is a showcase for the innovative and the ambitious. We invite all experts, builders, engineers, tinkerers, designers and dreamers to join us for an unforgettable two days.
+            The Ubuntu Summit is a showcase for the innovative and the ambitious. We invite all experts, builders, engineers, tinkerers, 
+            designers and dreamers to join us for an unforgettable two days.
           </p>
           <p>
             Tune in to our London livestream from the comfort of your home, from an Ubuntu release watch party, or from anywhere else around the world.
           </p>
           <div class="p-cta-block">
-            <a href="https://discourse.ubuntu.com/t/welcome-to-ubuntu-summit-25-10/65270" class="p-button--positive">Register for the Summit</a>
-            <a href="https://discourse.ubuntu.com/t/ubuntu-summit-25-10-timetable/65271" class="p-button">View the timetable</a>
+            <a href="https://discourse.ubuntu.com/t/register-for-ubuntu-summit/65270" class="p-button--positive">Register for the Summit</a>
+            <a href="https://ubuntu.com/summit/call-for-collaboration" class="p-button">Find out how you can contribute</a>
           </div>
         </div>
       </div>
     </div>
     <div class="u-fixed-width">
       <div class="p-section--shallow">
         <div class="u-embedded-media u-no-margin--bottom">
-          <iframe width="560"
-                  height="315"
+          <iframe width="560" 
+                  height="315" 
                   class="u-embedded-media__element"
-                  src="https://www.youtube.com/embed/aR-yTFtjGtw?si=VdJuwA6Uq4u2kEiV&autoplay=1&mute=1&loop=1&playlist=aR-yTFtjGtw"
-                  title="Ubuntu Summit 2024 Highlights"
-                  frameborder="0"
-                  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
-                  referrerpolicy="strict-origin-when-cross-origin"
-                  allowfullscreen></iframe>
+                  src="https://www.youtube.com/embed/gKMz7FVPJjo?si=2PPT29IJJ6KV59Ja" 
```

Comment:
you're missing some query parameters that the production page uses; namely `autoplay`, `mute`, and `loop` - could you add those so the video here behaves the same as the one on production (unless asked otherwise in the ticket)